lib/rules/require-padding-newlines-after-blocks.js

```diff
@@ -0,0 +1,128 @@
+/**
+ * Requires newline after blocks
+ *
+ * Type: `Boolean`
+ *
+ * Values: `true`
+ *
+ * #### Example
+ *
+ * ```js
+ * "requirePaddingNewLinesAfterBlocks": true
+ * ```
+ *
+ * ##### Valid
+ *
+ * ```js
+ * function () {
+ *     for (var i = 0; i < 2; i++) {
+ *         if (true) {
+ *             return false;
+ *         }
+ *
+ *         continue;
+ *     }
+ *
+ *     var obj = {
+ *         foo: function() {
+ *             return 1;
+ *         },
+ *
+ *         bar: function() {
+ *             return 2;
+ *         }
+ *     };
+ * }
+ * ```
+ *
+ * ##### Invalid
+ *
+ * ```js
+ * function () {
+ *     for (var i = 0; i < 2; i++) {
+ *         if (true) {
+ *             return false;
+ *         }
+ *         continue;
+ *     }
+ *     var obj = {
+ *         foo: function() {
+ *             return 1;
+ *         },
+ *         bar: function() {
+ *             return 2;
+ *         }
+ *     };
+ * }
+ * ```
+ */
+
+var assert = require('assert');
+
+var excludes = [
```

Comment:
Since TryStatement is in there twice, if I make it into an object I would have to have this:

```
var excludes = {
    'IfStatement': ['else'],
    'DoWhileStatement': ['while'],
    'TryStatement': ['catch', 'finally'],
    'CatchClause': ['finally'],
    'FunctionExpression': ['.'],
};
```

and then my check would be:
```
if (excludes[parentNode.type].indexOf(nextToken.value) !== -1) {
                    return;
                }
```

I'm okay with this, are you?